app/scripts/controllers/editBrands.js

```diff
@@ -8,24 +8,36 @@
  * Controller of the endslaverynowApp
  */
 angular.module('endslaverynowApp').controller('EditBrandsCtrl', [
-	'$firebaseObject',
+	'$firebaseArray',
 	'$routeParams',
 	'$scope',
-	function ($firebaseObject, $routeParams, $scope) {
+	function ($firebaseArray, $routeParams, $scope) {
 		$scope.loaded = false
-		$scope.allBrands = []
-		$scope.categories = []
+		$scope.brands = []
 
 		/* firebase */
-		var ref = firebase.database().ref()
-		var syncObject = $firebaseObject(ref)
+    var ref = firebase.database().ref('brands')
+    $scope.brands = $firebaseArray(ref)
 
-		syncObject.$loaded().then(function () {
-			for (var brand in syncObject.brands) {
-				$scope.allBrands.push(syncObject.brands[brand])
-			}
-			$scope.categories = syncObject.categories
+    $scope.deleteBrand = function(brandsRef, brand) {
+      var brandName = brand.name
+      var prompt = "Are you sure you want to delete brand '" + brandName + "'?"
 
+      if (!confirm(prompt)) {
+        return
+      }
+      brandsRef.$remove(brand).then(
+        function() {
+          var successMsg = "Successfully deleted brand '" + brandName + "'"
+          window.alert(successMsg)
```

Comment:
@aashnisshah I had done this for deleting categories, but now I'm thinking it'd be better user experience to just return to the list of categories/brands (like it does after this alert). What do you think would be best?

Reply:
I agree - better user experience if we return back to an updated list of categories/brands